wrappers/python/readme.md

```diff
@@ -35,21 +35,26 @@ Windows users can install the RealSense SDK 2.0 from the release tab to get pre-
   * **Note:** Use `sudo apt-get dist-upgrade`, instead of `sudo apt-get upgrade`, in case you have an older Ubuntu 14.04 version
 2. Install Python and its development files via apt-get
   * `sudo apt-get install python3 python3-dev`
-3. Run the top level CMake command with the following additional flag `-DBUILD_PYTHON_BINDINGS:bool=true`:
+3. Clone the librealsense repository and navigate into the directory:
+   * `git clone https://github.com/IntelRealSense/librealsense.git`
+   * `cd librealsense`
+4. Run the top level CMake command with the following additional flag `-DBUILD_PYTHON_BINDINGS:bool=true`:
```

Comment:
Let's call it configure and make and remove the cmake flag info as we have it later